tests/filecheck/mlir-conversion/with-mlir/dialects/pdl_interp/pdl_interp_ops.mlir

```diff
@@ -107,13 +108,14 @@ module @rewriters {
 // CHECK-NEXT:     builtin.module @rewriters {
 // CHECK-NEXT:       pdl_interp.func @pdl_generated_rewriter(%arg0 : !pdl.value, %arg1 : !pdl.value, %arg2 : !pdl.type, %arg3 : !pdl.value, %arg4 : !pdl.operation) {
 // CHECK-NEXT:         %0 = pdl_interp.create_attribute 10 : i64
-// CHECK-NEXT:         %1 = pdl_interp.create_operation "arith.subi"(%arg0, %arg1 : !pdl.value, !pdl.value) {"attrA" = %0} -> (%arg2 : !pdl.type)
-// CHECK-NEXT:         %2 = pdl_interp.get_result 0 of %1
-// CHECK-NEXT:         %3 = pdl_interp.create_operation "arith.addi"(%arg3, %2 : !pdl.value, !pdl.value) -> (%arg2 : !pdl.type)
-// CHECK-NEXT:         %4 = pdl_interp.get_result 0 of %3
-// CHECK-NEXT:         %5 = pdl_interp.get_results of %3 : !pdl.range<value>
-// CHECK-NEXT:         %6 = pdl_interp.get_results 0 of %3 : !pdl.range<value>
-// CHECK-NEXT:         pdl_interp.replace %arg4 with (%5 : !pdl.range<value>)
+// CHECK-NEXT:         %1 = pdl_interp.create_operation "arith.subi"(%arg0, %arg1 : !pdl.value, !pdl.value) {"attrA" = %0}  -> (%arg2 : !pdl.type)
```

Comment:
why is there an extra space after the attributes?

Reply:
Yeah, I also thought filecheck was whitespace-sensitive?